docs/release.rst

```diff
@@ -47,6 +47,9 @@ If you don't have nox, you should either use ``pipx run nox`` instead, or use
 - Update tags (optional; if you skip this, the GitHub release makes a
     non-annotated tag for you)
     - ``git tag -a vX.Y.Z -m 'vX.Y.Z release'``.
+    - To ensure the new tag is consistent with the version number in the sources, run this command:
+          - ``git diff "$(grep ^__version__ pybind11/_version.py | cut -d'"' -f2 | sed 's/^/v/')"``
+          - There should be no diff.
```

Comment:
```suggestion
          - ``grep ^__version__ pybind11/_version.py``
          - This should print the version you just tagged.
```

Reply:
I pushed a slightly different wording before seeing this comment.
Give me a moment to try again.
Simpler is better I agree.

Reply:
Plus it's not great for the "working" path to produce no feedback that it worked. Your wording is fine.